target/linux/sunxi/image/cortexa53.mk

```diff
@@ -38,6 +38,14 @@ define Device/friendlyarm_nanopi-neo2
 endef
 TARGET_DEVICES += friendlyarm_nanopi-neo2
 
+define Device/librecomputer_tritium-h5
+  DEVICE_VENDOR := Libre Computer
+  DEVICE_MODEL := Tritium (H5)
```

Comment:
So, is "H5" an alternate name, part of the name, or a revision? In the first case it shouldn't be part of the device string/compatible, in the second and third case it shouldn't be put into brackets here ...

Reply:
`Tritium` the name of the device.
`H5` is the part of the device variant (could i call that <board version>? it is released at the same time but have other pin compatible SoC).

there is three versions and all of them are called 'Tritium':
 - one with `H2+` SoC (arm32)
 - one with `H3` SoC (arm32)
 - one with `H5` SoC (arm64)

maybe we use `DEVICE_VARIANT` for that?
there has to be a way to distinguish between that three boards in a tree.
i would be glad to introduce other boards too.